src/libecalc/presentation/yaml/mappers/component_mapper.py

```diff
@@ -327,13 +264,13 @@ def from_yaml_to_dto(self, data: Dict) -> dto.Installation:
             )
             for fuel_consumer in data.get(EcalcYamlKeywords.fuel_consumers, [])
         ]
-        venting_emitters = [
-            self.__venting_emitters_mapper.from_yaml_to_dto(
-                venting_emitters,
-                regularity=regularity,
-            )
-            for venting_emitters in data.get(EcalcYamlKeywords.installation_venting_emitters, [])
-        ]
+
+        venting_emitters = []
+        for venting_emitter in data.get(EcalcYamlKeywords.installation_venting_emitters, []):
+            try:
+                venting_emitters.append(YamlVentingEmitter(**venting_emitter))
```

Comment:
so, this is where I basically expect the `YamlVentingEmitter(**venting_emitter).to_domain()`, if that is possible to do in one go, or if factories etc are needed, to do it in successive steps. The yamlventingemitter is just sth we want to validate the yaml (syntax), and parse the data, before mapping to our internal domain model(s)...

Reply:
Are you sure we should do it here? If we are to avoid evaluating expressions in core/domain, we need to give a rate that is evaluated to domain. This means YamlVentingEmitter(**venting_emitter).to_dto(variables_map, regularity). Do we want to send variables_map to component_mapper?

Reply:
No! :) And that is why I gave an answer with reservations based on what is needed to be done. I think we should do the conversion of regularity and variables to what the core should have, BEFORE we convert to domain models. This can happen e.g. in a `factory` or request a `service` to do the conversion, as a part of the `to_domain()` ...

Reply:
In general I do not think to_domain should have any parameters